server/src/main/java/org/opensearch/cluster/coordination/NoClusterManagerBlockService.java

```diff
@@ -47,27 +47,27 @@
  * @opensearch.internal
  */
 public class NoClusterManagerBlockService {
-    public static final int NO_MASTER_BLOCK_ID = 2;
-    public static final ClusterBlock NO_MASTER_BLOCK_WRITES = new ClusterBlock(
-        NO_MASTER_BLOCK_ID,
+    public static final int NO_CLUSTER_MANAGER_BLOCK_ID = 2;
```

Comment:
```suggestion
    /** 
      * Ordinal ID used to block when no cluser manager is available
      * @opensearch.internal
      */
    public static final int NO_CLUSTER_MANAGER_BLOCK_ID = 2;
```

Of course feel free to update the javadoc to something more clear.

Reply:
@nknize Got it. Thanks a lot for your detailed explanation! In yesterday I didn't realize that you would like me to add `@opensearch.internal` annotation on the variable.
I've got a question, since the whole class has been marked with `@opensearch.internal`:
https://github.com/opensearch-project/OpenSearch/blob/00db112f32d947e758d7d5f6e4a6c7444a22a7c3/server/src/main/java/org/opensearch/cluster/coordination/NoClusterManagerBlockService.java#L47)
Is that necessary to mark the variable? or you just want to emphasize it?

Reply:
Ooh! Good point! No I think that covers it. Thx for pointing it out.

Reply:
@nknize No problem! Thank you for your suggestion on the action we should make to avoid the public API abuse, this will be a good example. 😄👍